### PR TITLE
Commit version 1 only at first

### PR DESCRIPTION
**Pull request by hzoo**
Showing off making a PR via zoom

> I should probably explain why I'm making the change in the description

### CODE REVIEW

File: rockpaperscissors.py [hunk 1]
```diff
@@ -77,126 +75,3 @@ def r3():
     r0()
 else:
     print('invalid entry, run script again')
-'''
-#version 2.0 May 5, 2020
```
Comment (zupanssi) [inline rockpaperscissors.py:81]: NOOOOOOO!!!!!!

Comment (hzoo) [reply rockpaperscissors.py:81]: And this is just me explaining we are adding the other file in the next commit

**Comment by hzoo** [comment]
Adding another commit to show a diff

**Comment by zupanssi** [comment]
JK, THANKS.

**Pull request merged**